x-pack/plugins/timeline/README.md

```diff
@@ -0,0 +1,11 @@
+# timeline
+This plugin provides a common timeline table component and supporting functionality for use in other plugins.
```

Comment:
@kqualters-elastic Thanks for putting this PR together. 

Since this is a top level plugin, it would be nice to have additional context. Could we add a description of what Timeline is? Perhaps we could also have a paragraph explaining when and how you could use this. For example we could say "A Timeline is a grid component with a server implementation that helps users investigate... etc" and then "If you want to include Timeline in your plugin you'll need to do..."

Also, do we need CODEOWNERS here? I think at least initially we should set it to someone in RAC.